translations/ui-orders/en.json

```diff
@@ -391,18 +391,7 @@
   "itemDetails.description": "Description",
   "itemDetails.edition": "Edition",
   "itemDetails.productId": "Product ID",
-  "itemDetails.productIdType.CODEN": "CODEN",
```

Comment:
Are you sure we don't need to translate these values anymore?

Reply:
Asked in Jira